app/src/main/java/com/everest/moviedb/model/Movie.kt

```diff
@@ -0,0 +1,31 @@
+package com.everest.moviedb.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Movie(
```

Comment:
Always to better to use different data models for network and UI layer.
Seems like only one is being used